packages/assets/src/tools/validate-files.ts

```diff
@@ -18,6 +19,33 @@ async function fromOrgFiles(org: ThemeVariant) {
   return {org, files: files.map(cleanFilenames)};
 }
 
+async function verifyThatMissingOrgFilesHasCommonReplacements(
+  missingOrgFiles: string[],
+) {
+  let filesMissingFromBothOrgAndCommonFolder = <string[]>[];
+
+  for (const missingOrgFile of missingOrgFiles) {
+    const fullPath = path.join(
+      __dirname,
+      '..',
+      '..',
+      'files',
+      'common',
+      missingOrgFile,
+    );
+    const commonFileExistsForMissingOrgFile = await promises
+      .access(fullPath, constants.F_OK)
```

Comment:
Maybe this bit is more readable with the `fs.`-prefixes included?

Reply:
Also, might also just be a small helper function reading `exists`. Could make it read easier.

Reply:
Considered using `fs.exists` but [it seems to be deprecated](https://nodejs.org/api/fs.html#fsexistspath-callback), so I guess we should not use it then?

`fs.existsSync` is [not deprecated](https://nodejs.org/api/fs.html#fsexistssyncpath), but I guess we shouldn't add synchronous code if we don't have to?

Reply:
Or are you proposing extracting this into a helper function ourselves, for example called `exists`? 😅

Reply:
Yep, that one. Extracting the block and call the function `exists`